File: beginner/pagina12/3039/3039.php
<?php 
    $n = (int) readline();
    $carrinhos = 0; $bonecas = 0;

    for($i = 0; $i < $n; $i++){
        list($nome, $sexo) = explode(" ", readline());

        if($sexo == "M"){ $carrinhos += 1;}
        else { $bonecas += 1;}
    }

    echo "$carrinhos carrinhos\n";
    echo "$bonecas bonecas\n";
?>

File: beginner/pagina12/3039/3039.rs
use std::io;

fn main(){

    let mut input = String::new();

    io:;stdin().read_line(&mut input).unwrap();

    let N: usize = input.trim().parse::<usize>().unwrap_or_default();

    let mut meninos: u32 = 0;
    let mut meninas: u32 = 0;

    for _ in 0..N {

        input.clear();
        io:;stdin().read_line(&mut input).unwrap();

        let data: Vec<&str> = input.trim().split_whitespace().collect();
        let genero: &str = data[1];

        if genero == 'M' {
            meninos += 1;
        } else {
            meninas += 1;
        }

    }

    println!("{} carrinhos", meninos);
    println!("{} bonecas", meninas);
}